src/core/secure/keychain.ts

```diff
@@ -90,7 +91,8 @@ export const verifyPinCode = async (pinCode: string): Promise<boolean> => {
         const encryptionKey = await getEncryptionKey(pinCode);
         await readEncrypted(KEY_PIN_SAMPLE, encryptionKey);
         return true;
-    } catch (e) {
+    } catch (err) {
+        Sentry.captureException(new Error(JSON.stringify(err))); // needed?
```

Comment:
this would be ok, but if we can filter keystore errors, we don't want to know that a user mistyped his pin code. So i think we can remove this.